modules/sdk-coin-icp/src/lib/unsignedTransactionBuilder.ts

```diff
@@ -21,7 +21,8 @@ export class UnsignedTransactionBuilder {
   }
 
   async getUnsignedTransaction(): Promise<PayloadsData> {
-    const interval = MAX_INGRESS_TTL - PERMITTED_DRIFT - 120 * 1000_000_000; // 120 seconds in milliseconds
+    // interval value can be altered here to change the count of ingressExpiries which eventually changes the number of signatures required
```

Comment:
number of signature gives a wrong idea.
```suggestion
    // interval value can be altered here to change the count of ingressExpiries which eventually changes the number of payloads that needs to be signed
```

Reply:
changed